google/resource_google_project.go

```diff
@@ -134,6 +256,34 @@ func resourceGoogleProjectCreate(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 
+	// set up App Engine, too
+	app, err := expandAppEngineApp(d)
+	if err != nil {
+		return err
+	}
+	log.Printf("[DEBUG] Creating App Engine if %v is true", app != nil)
```

Comment:
This one might be a little chatty - since we already have "Enabling App Engine", the lack of that tells us everything that this statement tells us.

Reply:
Oops, sorry! This was a debug message from when I was figuring out errors. I'll remove it.